packages/module/patternfly-docs/content/extensions/data-view/examples/DataView/DataView.md

```diff
@@ -5,23 +5,31 @@ section: extensions
 subsection: Data view
 # Sidenav secondary level section
 # should be the same for all markdown files
-id: Data view
+id: Data view layout
 # Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
 source: react
 # If you use typescript, the name of the interface to display props for
 # These are found through the sourceProps function provided in patternfly-docs.source.js
 propComponents: ['DataView']
 sourceLink: https://github.com/patternfly/react-data-view/blob/main/packages/module/patternfly-docs/content/extensions/data-view/examples/DataView/DataView.md
 ---
-
 import DataView from '@patternfly/react-data-view/dist/dynamic/DataView';
+import DataViewToolbar from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
 
 The **data view** component renders record data in a configured layout.
 
-### Basic example
+### Layout example
+
+Data view is expected to consist of header, data part and footer stacked below each other and passed as `children`.
+
+```js file="./DataViewLayoutExample.tsx"
+
+```
+
+### Predefined layout components
 
-A blank example of the data view layout.
+You can make use of the predefined layout components to display a default header and footer. See [data view toolbar](/data-view/data-view-toolbar) for more information 
```

Comment:
I also see the a11y tests failing there with `link-in-text-block`. The link is added the same way as in the older docs framework version. Did anything change with v6?

Reply:
You are not the only person who's run into this error during the migration to the v6 alphas. I will have to investigate how others have resolved this issue.

Reply:
I think the recommended solution (since the rule being flagged by the a11y tests is a 'new experimental rule') is to ignore it.
You can see that being done in another extension here: https://github.com/patternfly/react-catalog-view/pull/54/commits/bd81ee6fb18d6d21533ce344fd84c1252a701632

Reply:
Cool, thank you 🙂